pages.fr/sunos/svcadm.md

```diff
@@ -0,0 +1,24 @@
+# svcadm
+
+> Manipuler les instances de service.
+> Plus d'information : <https://www.unix.com/man-page/linux/1m/svcadm>.
+
+- Activer un service dans la base de données de service:
+
+`svcadm enable {{service_name}}`
```

Comment:
Please translate the placeholders, like `service_name`, etc., for get this easier for non-English speakers.